test/gurka/test_ferry_connections.cc

```diff
@@ -119,12 +119,15 @@ TEST(Standalone, ShortFerry) {
 }
 
 TEST(Standalone, TruckFerryDuration) {
-  // corresponds to 33.3 m/sec (120 km/h) in the below map
-  uint32_t ferry_secs = 9;
+  // corresponds to 21.4 m/sec (~77 km/h) in the below map
```

Comment:
Changes to this test are necessary, since ferry route speeds are clamped by this new `kMaxAssumedTruckSpeed` value as well.

If anyone's wondering about how fast vehicle carrying ferries can go, I just went down that rabbit hole: The HSC Francisco running between Buenos Aires and Montevideo holds the world record for the fastest ferry at ~107 kph.[^1] However, that's the top speed and at an average route duration of 2h30mins for ~220 km [^2], even that would not be clamped.

[^1]: https://www.guinnessworldrecords.com/world-records/505447-fastest-ferry
[^2]: https://es.wikipedia.org/wiki/Francisco_(barco)